.travis.yml

```diff
@@ -8,13 +8,13 @@ services:
 matrix:
   include:
     - php: 7.1
-      env: DB=MYSQL PHPUNIT_TEST=1
+      env: DB=MYSQL RECIPE_VERSION=4.x-dev PHPUNIT_TEST=1 PHPCS_TEST=1
```

Comment:
It used to be `4.6.x-dev`, and not `4.x-dev`

Reply:
Updated to `4.6.x-dev`